deployment/docker-compose/litellm_proxy_config.yaml

```diff
@@ -16,17 +16,20 @@ model_list:
     litellm_params:
       model: gpt-4o
       api_key: "os.environ/OPENAI_API_KEY"
-      # model: gemini/gemini-1.5-pro-latest
-      # api_key: "os.environ/GEMINI_API_KEY"
-      # safety_settings:
-      #   - category: HARM_CATEGORY_HARASSMENT
-      #     threshold: BLOCK_NONE
-      #   - category: HARM_CATEGORY_HATE_SPEECH
-      #     threshold: BLOCK_NONE
-      #   - category: HARM_CATEGORY_SEXUALLY_EXPLICIT
-      #     threshold: BLOCK_NONE
-      #   - category: HARM_CATEGORY_DANGEROUS_CONTENT
-      #     threshold: BLOCK_NONE
+  - model_name: generate-gemini-response
+    litellm_params:
+      # Set VERTEXAI_ENDPOINT environment variable or direclty enter the value:
+      api_base: "os.environ/VERTEXAI_ENDPOINT"
+      model: vertex_ai/gemini-1.5-pro
+      safety_settings:
+        - category: HARM_CATEGORY_HARASSMENT
+          threshold: BLOCK_ONLY_HIGH
```

Comment:
We want BLOCK_NONE, but still haven't been added to the allow list..

Reply:
What happens without it? Doesn't OpenAI also block some content?

Reply:
Last time I experimented, it would block our safety filter because of the few shot examples.. we should experiment properly after merging this!